sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_models.py

```diff
@@ -87,55 +87,55 @@ def key_id(self):
 
     @property
     def enabled(self):
-        # type: () -> bool
+        # type: () -> Union[bool, None]
```

Comment:
Union[T, None] == Optional[T]

Reply:
I'll update it, and in all other places.